content/en/tracing/profiling/java.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Profiling Java Applications
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+The Datadog Profiling library supports the Java JRE 11 and higher of both Oracle JDK and OpenJDK. Early-access versions of Java aren't supported.
+Java profiling and tracing are shipped within the same `dd-java-agent.jar` library. In order to use tracing or profiling, you must enable these functions. To enable tracing, follow the [enable tracing documentation][1].
+
+**Note**: Some specific builds will work with Datadog profiling if they support [Java Flight Recorder][2].
```

Comment:
Do we need this?

Reply:
Is Java flight recorder a requirement of the profiling lib. Or "some specific builds" refers to some non-vanilla version of java

Reply:
Yes we need flight recorder, but I don't think we should say so in an intro tutorial; it won't mean anything to most users. I don't know what "some specific builds" refers to so I think we should remove it :-)

Reply:
added the sentence on top